tftp/tftp.go

```diff
@@ -17,6 +17,7 @@ var tftpFiles = map[string][]byte{
 	"snp-nolacp.efi": ipxe.MustAsset("bin/snp-nolacp.efi"),
 	"ipxe.efi":       ipxe.MustAsset("bin/ipxe.efi"),
 	"snp-hua.efi":    ipxe.MustAsset("bin/snp-hua.efi"),
```

Comment:
this one is not in definition of any of the make files.  So locally I had to modify my make file to ignore this target for creating the bindata.go

Reply:
yeah snp-hua.efi is basically in an unreproducible state. We have one build that works, builds are super finnicky and we try not to touch it :|